.github/workflows/integration.yml

```diff
@@ -1,35 +1,50 @@
 name: Integration
 on: [push, pull_request]
 
-env:
-  GO_VERSION: 1.16
-
 jobs:
   integration:
+    strategy:
+      matrix:
+        go: [1.16, 1.17, 1.18]
```

Comment:
Testing on all three versions will take longer and cost more. Do you think passing on Go 1.16 is likely to fail on Go 1.17 or Go 1.18?

Reply:
I thought GitHub was pretty generous with the open source use of github actions.

Reply:
99designs has exceeded the free tier, but it's unclear to me as to how much this repository contributed to that usage, compared to their other open source repositories, or their private GitHub action usage. Since it's not my money, I try to only use what I need. I think your revision where you are testing oldest supported (Go 1.16) and newest supported (Go 1.18) is a great compromise. Thanks!

Reply:
I would prefer the last 2 major versions of Go, to align with the Go EOL policy

Reply:
@mtibben I agree with you that is the goal, but we at Khan are using Go 1.16 until [GCP AppEngine](https://cloud.google.com/appengine/docs/standard/go/release-notes) supports a newer Go version, and I can't help maintain this project if it is beyond that. GCP tells us they plan on skipping Go 1.17 and releasing support for Go 1.18 in H2 of 2022 (so July-Dec 2022) but haven't increased that. As a result, we previously agreed to keep gqlgen supporting the same Go version as Khan uses in prod (and newer).

Reply:
yeah fair enough 👍